test/addons/dlopen-ping-pong/test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../../common');
+
+if (common.isWindows) {
+  common.skip('dlopen global symbol loading is not supported on this os.');
+  return;
+}
+
+if (common.isAIX) {
+  common.skip('this test does not pass on AIX.');
+  return;
+}
+
+const assert = require('assert');
+const path = require('path');
+const os = require('os');
+
+const bindingPath = require.resolve(`./build/${common.buildType}/binding`);
+process.dlopen(module, bindingPath,
+               os.constants.dlopen.RTLD_NOW | os.constants.dlopen.RTLD_GLOBAL);
```

Comment:
These variables are defined in an `if` macro guard. If they are not available in the current OS, then this test should be skipped, right?

Reply:
I guess so.